R/spi.R

```diff
@@ -249,9 +253,10 @@ spi.get_predicted <- function(x, ...) {
 
 
   # lower bound
-  if (k == 1) {
+  if (!is.na(k) && all(k == 1)) {
     x1 <- l
     w.l <- 1
+    x.l <- NA
```

Comment:
@bwiernik There are some edge cases, where this if-statement is TRUE, and then `x.l` was not initialized. I set it to `NA`, is that ok?